.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
```

Comment:
As above, this isn't referenced in the module, so probably isn't needed

Reply:
As mentioned above, this content is added automatically as part of `eslint --init` boilerplate. I can remove if you like, however.